minorminer/utils/raster_embedding.py

```diff
@@ -117,27 +117,42 @@ def find_multiple_embeddings(S, T, timeout=10, max_num_emb=float('inf')):
             Defaults to 10.
         max_num_emb (int, optional): Maximum number of embeddings to find.
             Defaults to inf (unbounded).
+        inplace (bool, optional): Specifies whether to remove visited nodes 
```

Comment:
Specifies whether to create a copy of the target graph. Defaults to False, so that the input argument is unchanged. When True, nodes may be deleted from the target graph as part of the iterative search process.